src/test/java/com/google/api/codegen/testdata/nodejs_main_no_path_templates.baseline

```diff
@@ -0,0 +1,165 @@
+============== file: lib/no_templates_service_api.js ==============
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * EDITING INSTRUCTIONS
+ * This file was generated from the file
+ * https://github.com/googleapis/googleapis/blob/master/library.proto,
+ * and updates to that file get reflected here through a refresh process.
+ * For the short term, the refresh process will only be runnable by Google
+ * engineers.
+ *
+ * The only allowed edits are to method and file documentation. A 3-way
+ * merge preserves those additions if the generated source changes.
+ */
+'use strict';
+
+var arguejs = require('arguejs');
+var configData = require('./no_templates_service_client_config');
+var gax = require('google-gax');
+
+/** The default address of the service. */
+var SERVICE_ADDRESS = 'no-path-templates.googleapis.com';
+
+/** The default port of the service. */
+var DEFAULT_SERVICE_PORT = 443;
+
+var CODE_GEN_NAME_VERSION = 'gapic/0.1.0';
+
+var DEFAULT_TIMEOUT = 30;
+
+/**
+ * The scopes needed to make gRPC calls to all of the methods defined in
+ * this service.
+ */
+var ALL_SCOPES = [
+];
+
+/**
+ * @class
+ * @param {?Object} opts - The optional parameters.
+ * @param {String} opts.servicePath
+ *   The domain name of the API remote host.
+ * @param {number} opts.port
+ *   The port on which to connect to the remote host.
+ * @param {Function} opts.getCredentials
+ *   Custom function to obtain the credentials.
+ * @param {grpc.ClientCredentials} opts.sslCreds
+ *   A ClientCredentials for use with an SSL-enabled channel.
+ * @param {Object} opts.grpc
+ *   When specified, this is used as the grpc module.
```

Comment:
Could you clarify this description to state that that if this is not specified, then the `grpc` module is loaded automatically?